lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -351,59 +317,50 @@ FUNCTIONKIND_VALUES(VALUE)
 
         library->AddMember(installTarget, methodPropID, func);
 
-        // do extra logic here which didnt easily fit into the macro table
-        switch (funcKind)
+        // Make function available to other internal facilities that need it
+        // applicable for specific functions only - this may need review upon moving other functions into JsBuiltins
+        if (current == JsBuiltInFile::Array_prototype)
```

Comment:
This logic supports several internal re-uses of certain Array functions and Math.max and Math.min. I wanted to get rid of it but some of them appear needed and others would be quite a bit of work.